next/.env

```diff
@@ -12,3 +12,11 @@ NEXT_PUBLIC_AWS_REGION=eu-central-1
 NEXT_PUBLIC_FORMS_URL=https://nest-forms-backend.dev.bratislava.sk
 NEXT_PUBLIC_CITY_ACCOUNT_URL=https://nest-city-account.staging.bratislava.sk
 FOP_URL=https://fop.dev.bratislava.sk
+
+NEXT_PUBLIC_MINIO_ACCESS_KEY=FORMS_DEV
```

Comment:
if these are used from backend side on next, they shouldn't be "NEXT_PUBLIC_..." prefixed. Everything prefixed this way is readable from browser.

If these are going to be different accross environments, use the `bratiska-cli-build.environment.env` files - this one is jsut for those which are the same everywhere. Moreover, if the values shouldn't be publicly accessible, these should be in secrets - in that case, leave it up to @skifahrer to set it up.

If you think some subset of this (port, use ssl) will be always the same, even for development, I'd prefer placing that as hardocded directly into code and keeping only those that change in the env vars.

Reply:
fixed like we talked